vcbuild.bat

```diff
@@ -39,6 +39,7 @@ set enable_vtune_arg=
 set configure_flags=
 set build_addons=
 set dll=
+set ci_test_targets="addons debugger doctool inspector known_issues message parallel sequential"
```

Comment:
This will include the quotes in the variable. Open the quotes before the variable name instead:

```bat
set "ci_test_targets=addons debugger doctool inspector known_issues message parallel sequential"
```